cms/djangoapps/contentstore/course_group_config.py

```diff
@@ -111,9 +111,18 @@ def _get_usage_info(course, unit, item, usage_info, group_id, scheme_name=None):
         """
         Get usage info for unit/module.
         """
+        if unit.category == 'sequential' or item.category == 'split_test':
```

Comment:
If possible, I would really like to find a way to avoid looking specifically at "split_test". There are other XBlocks that can have children, such as the conditional XBlock. 

And it's very odd that a "unit" would be of type "sequential" or "split_test". Is "unit" just the parent of "item"?